examples/howto/using_password_input.py

```diff
@@ -0,0 +1,32 @@
+from bokeh.models.widgets import PasswordInput, TextInput, PreText, Button
+from bokeh.layouts import column, row
+from bokeh.plotting import show, output_file
+from bokeh.models.callbacks import CustomJS
+
+# Set User and Password:
+USER = "Mau"
+PASSWD = "Bok3h"
+
+## Create Texts, inputs and button
+text = PreText(text="LOGIN TO KNOW\nTHE SECRET:")
+user = TextInput(placeholder="username", title="(UserName: "+USER+")")
+pwd = PasswordInput(placeholder="password", title="(Password: "+PASSWD+")")
+btn = Button(label="GO!",width=150)
```

Comment:
`# Set User and Password:` and `## Create Texts, inputs and button` both seem redundant to me